### PR TITLE
Update Run a Node with volume creation and pruning instructions

### PR DESCRIPTION
**Pull request by mapachurro**
Users have been getting stuck on the step regarding creating a specific volume size. This doesn't have to do with geth, it's an OS / devops level concern. 

This PR provides a brief overview and steps to follow.

### CODE REVIEW

File: docs/build-on-linea/volume-creation.mdx [hunk 1]
```diff
@@ -0,0 +1,88 @@
+:::note
```
Comment (jlwllmr) [inline docs/build-on-linea/volume-creation.mdx:1]: "Admonitions", as Docusaurus calls them, need empty lines around them. So I think it should be:

:::note

[content]

:::

https://docusaurus.io/docs/markdown-features/admonitions

**Review by jlwllmr — DISMISSED** [review]
I think we need to resolve the comment in `volume-creation.mdx`

File: docs/build-on-linea/volume-creation.mdx (hunk 1 above)
Comment (mapachurro) [reply docs/build-on-linea/volume-creation.mdx:1]: Updated. Thank you for your detailed notes!

**Review by WenLanBan — APPROVED** [review]
looks good

**Pull request merged**